src/components/ButtonCreateTask/ButtonCreateTask.tsx

```diff
@@ -1,14 +1,19 @@
-// import { useState } from "react"
+import { useAppSelector } from "../../redux/hooks"
+import { Task } from "../../types/Task"
+import "./style.css"
 
 type Props = { onShowModal: () => void }
 
 export function ButtonCreateTask({ onShowModal }: Props) {
-  // const [modal, setModal] = useState(0)
-
+  const tasks: Task[] = useAppSelector((state) => state.allTasks.tasks)
```

Comment:
Is the `: Task[]` part here needed? What is the type if you leave that extra type definition off?

Reply:
Nothing happens if I take that type off! Woot!